presto-tests/src/test/java/com/facebook/presto/tests/TestTpchDistributedStats.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests;
+
+import com.facebook.presto.tests.statistics.StatisticsAssertion;
+import com.facebook.presto.tpch.ColumnNaming;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Range;
+import io.airlift.tpch.TpchTable;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.tests.statistics.Metric.OUTPUT_ROW_COUNT;
+import static com.facebook.presto.tests.statistics.MetricComparisonStrategies.absoluteError;
+import static com.facebook.presto.tests.statistics.MetricComparisonStrategies.defaultTolerance;
+import static com.facebook.presto.tests.statistics.MetricComparisonStrategies.noError;
+import static com.facebook.presto.tests.statistics.MetricComparisonStrategies.relativeError;
+import static com.facebook.presto.tests.tpch.TpchQueryRunner.createQueryRunnerWithoutCatalogs;
+import static java.util.Collections.emptyMap;
+
+public class TestTpchDistributedStats
+{
+    private final StatisticsAssertion statisticsAssertion;
+
+    public TestTpchDistributedStats()
+            throws Exception
+    {
+        DistributedQueryRunner runner = createQueryRunnerWithoutCatalogs(emptyMap(), emptyMap());
+        runner.createCatalog(
+                "tpch",
+                "tpch",
+                ImmutableMap.of("tpch.column-naming", ColumnNaming.STANDARD.name()));
+        statisticsAssertion = new StatisticsAssertion(runner);
+    }
+
+    @Test
+    void testTableScanStats()
+    {
+        TpchTable.getTables()
+                .forEach(table -> statisticsAssertion.check("SELECT * FROM " + table.getTableName())
+                        // TODO use noError
+                        .estimate(OUTPUT_ROW_COUNT, defaultTolerance()));
+    }
+
+    @Test
+    void testFilter()
```

Comment:
By convention, test methods are `public`

Reply:
yeah... let it be. However as you can see `public` is redundant.